clang/lib/CIR/CodeGen/CIRGenDecl.cpp

```diff
@@ -59,6 +54,8 @@ CIRGenFunction::buildAutoVarAlloca(const VarDecl &D) {
 
   Address address = Address::invalid();
   Address allocaAddr = Address::invalid();
+  // TODO[OpenMP]: Set openMPLocalAddr with the equivalent of
+  // getOpenMPRuntime().getAddressOfLocalVariable().
```

Comment:
Can we get these as part of more specific `Unimplemented::`? Example `UnimplementedFeature::openMPLocalAddr()` or something that conveys the intent? This should make it easier for others in the community to pick up work and help out!